graders/python/python_autograder/pl_execute.py

```diff
@@ -198,16 +231,16 @@ def execute_code(
 
     student_result = {}
     for name in names_from_user:
-        student_result[name] = student_code.get(name, None)
+        student_result[name] = student_globals.get(name, None)
 
     plot_value = None
     if include_plt:
-        for key in list(student_code):
+        for key in list(student_globals):
```

Comment:
```suggestion
        for key, global in student_globals.items():
```
It looks like the value is always looked up with it's key? Wasted work if we just loop through instead.

Reply:
Changed in 52d3263. `global` is a reserved word AFAICT, and the key actually wasn't used in the loop body at all so I just went with `student_globals.values()`.